assets/src/components/subway_status/lcd_subway_status.tsx

```diff
@@ -293,33 +124,20 @@ const convertLocationlessExtendedAlertToContracted = (
     ? { type: "contracted", alerts: [section.alert] }
     : section;
 
-/**
- * Determines whether we should show the last rule, below the Green Line section.
- *
- * We show the last rule as long as the last pill in the pill column is no lower
- * than where it would be if all statuses were normal.
- *
- * This is the case when the two following conditions are met:
- * - The first 3 sections (BL, OL, RL) are contracted with 1 alert (or normal status)
- * - The GL section is *either*:
- *   - extended, *or*
- *   - contracted without a separate pill on its second alert (or just no second alert)
- *
- * In all other cases, we do not show the last rule.
- */
-const shouldShowLastRule = ({ blue, orange, red, green }: SubwayStatusData) => {
-  const firstThree = [blue, orange, red];
-
-  const firstThreeContracted = firstThree.every(isContractedWith1Alert);
-  const glExtended = isExtended(green);
-  const glContractedWithNoSecondPill =
-    isContracted(green) && !green.alerts?.[1]?.route_pill;
-
-  return firstThreeContracted && (glExtended || glContractedWithNoSecondPill);
-};
-
 const isExtendedWithNoLocation = (
   section: Section,
 ): section is ExtendedSection => isExtended(section) && !section.alert.location;
 
+const showHeader = (sections: Section[]) => {
```

Comment:
I like this function, but might want to make note of it in a comment in the CSS file. Since the values (120, 216, 160) are the sum of several values, I feel like it would be more painful to tie this to constants shared between CSS and JS and am fine with having it hardcoded like this